ui/ui-components/components/Navigation.tsx

```diff
@@ -177,7 +177,7 @@ export default function Navigation(props) {
           </span>
         </div>
 
-        {!navigationMode.includes("unauthenticated") && (
+        {!navigationMode?.includes("unauthenticated") && (
```

Comment:
These were causing the app to crash when there was an error. However, with the hydration error issues resolved this may no longer be a problem.